csi/moac/pool.ts

```diff
@@ -86,7 +94,7 @@ class Pool {
   //
   // @param {object[]} replicas   New list of replicas properties for the pool.
   //
-  _mergeReplicas (replicas) {
+  _mergeReplicas(replicas: any[]) {
```

Comment:
replica: Replica is used below so shouldn't this be replicas: Replica[]?

Reply:
The same comment here. Those are not replica objects. replicasProperties might be a better name but too long :-)